packages/zen-bull-netting/test/unit-test/CancelNonce.t.sol

```diff
@@ -0,0 +1,33 @@
+pragma solidity ^0.8.13;
+
+pragma abicoder v2;
+
+// test dependency
+import { console } from "forge-std/console.sol";
+import { ZenBullNettingBaseSetup } from "../ZenBullNettingBaseSetup.t.sol";
+
+/**
+ * Unit tests
+ */
+contract CancelNonce is ZenBullNettingBaseSetup {
+    uint256 public traderPk;
+    address public trader1;
+
+    function setUp() public override {
+        ZenBullNettingBaseSetup.setUp();
+
+        traderPk = 0xA12CD;
+        trader1 = vm.addr(traderPk);
+
+        vm.label(trader1, "trader1");
+    }
+
+    function testCancelNonce() public {
```

Comment:
Maybe better to test in the context of an order - ie flip a nonce to false and the auction fill should fail, but maybe that is tested already.